resources.whatwg.org/build/README.md

```diff
@@ -12,19 +12,46 @@ The `deploy.sh` script is used by most WHATWG standards and is meant to run eith
 - Running the [Nu HTML checker](http://checker.html5.org/) on the build results
 - Deploying the build results to the WHATWG web server
 
-For non-local deploys, it is dependent on the following environment setup:
+For Travis CI deploys, it is dependent on the following environment setup:
 
 - `deploy_key.enc` must contain a SSH private key, [encrypted for Travis](https://docs.travis-ci.com/user/encrypting-files/) for the appropriate repository.
 - The environment variable `$ENCRYPTION_LABEL` must contain the encryption label produced by the Travis encryption process.
 
-Optional environment variables:
+For GitHub Actions deploys, the following environment variables are required:
 - `$SERVER` is the server to deploy to.
 - `$SERVER_PUBLIC_KEY` is the public key of the deploy server, in the format of `known_hosts`.
+- `$SERVER_DEPLOY_KEY` is the deploy key for the server, which will be passed to `ssh-add`.
+
+Optional environment variables:
 - `$EXTRA_FILES` are extra files to copy for each build. Shell wildcards are allowed, and directory structure will be preserved. Example: `EXTRA_FILES="images/*.png"`.
 - `$POST_BUILD_STEP` is an extra step to run after each build. Evaluated with the `$DIR` variable set to the build directory. Example: `POST_BUILD_STEP='python generate-stuff.py "$DIR"'`.
 
 To cause particular errors or warnings emitted by the HTML checker to be suppressed, add a file named `.htmlcheckerfilter` at the root of the repo for a particular standard, and put filter patterns into it, as documented at https://github.com/validator/validator/wiki/Message-filtering#using-the---filterfile-option.
 
+An example `.github/workflows/deploy.yml` file that uses this script would then be as follows:
+
+```yaml
+name: deploy
+on:
+  push:
+    branches:
+      - master
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
```

Comment:
We've been using Debian elsewhere; any particular reason to go with Ubuntu here?

Reply:
https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources is the set of GitHub-provided runners. We could provide our own Docker image and lock everything down, but I actually think it's not necessary for the relatively plain dependencies that we have.